log/logger.go

```diff
@@ -1,5 +1,14 @@
 // Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
+//
+// This file is a derived work, based on the go-ethereum library whose original
+// notices appear below.
+//
+// It is distributed under a license compatible with the licensing terms of the
+// original code from which it is derived.
+//
+// Much love to the original authors for their work.
+// **********
```

Comment:
I found a whole bunch of these files in subnet-evm when I was linting them, in which there just was not an original license and I found it kinda annoying (I added the geth license, but then Cey had me remove it as for for whatever reason it wasn't there originally).

Too much for this PR but I would if there could be some beter way to denote these files